Remotion/ObjectBinding/Web.UnitTests/UI/Controls/BocListImplementation/Rendering/BocSimpleColumnRendererTest.cs

```diff
@@ -46,15 +46,15 @@ public override void SetUp ()
       Column.ColumnTitle = "FirstColumn";
       Column.PropertyPathIdentifier = "DisplayName";
       Column.FormatString = "unusedWithReferenceValue";
-      Column.OwnerControl = List;
+      Column.OwnerControl = List?.Object;
```

Comment:
The null-propagation is needed, because the value of `List` on line 49 is `null`. `List` gets its value on line 51 in the `base.SetUp` method.

Reply:
ah, okay, that's...not nice. 
two options: remove the line completly since it's just wrong or fix it by moving it after the base-call. we should try to first move it after the base call and check if the tests still work since that is what was originally intended.